web/files/references.go

```diff
@@ -28,10 +50,21 @@ func listReferencesHandler(c echo.Context) error {
 		return err
 	}
 
-	req := &couchdb.ViewRequest{
-		Key: []string{doctype, id},
+	key := []string{doctype, id}
+	reqCount := &couchdb.ViewRequest{Key: key, Reduce: true}
+
+	var resCount couchdb.ViewResponse
+	err = couchdb.ExecView(instance, consts.FilesReferencedByView, reqCount, &resCount)
+	if err != nil {
+		return err
 	}
 
+	count := 0
+	if len(resCount.Rows) > 0 {
+		count = int(resCount.Rows[0].Value.(float64))
```

Comment:
Yes, Row.Value is now an interface{} (we had to change it from your initial view implementation to support other view results).